roaringbitmap/src/test/java/org/roaringbitmap/RandomisedTestData.java

```diff
@@ -25,7 +25,7 @@ public static RoaringBitmap randomBitmap(int maxKeys, double rleLimit, double de
               final IntStream stream;
               if (choice < rleLimit) {
                 stream = rleRegion();
-              } else if (choice < denseLimit) {
+              } else if (choice < rleLimit+ denseLimit) {
```

Comment:
No because on lines 40 and 46 the rleLimit is chosen as the lower bound for the value of the dense limit, but this can be changed to make things clearer.

Reply:
Ok, got it. We think of container as sparse or dense. And RLE is just a subtype of dense.

Reply:
Maybe change the word "dense" to "dirty" here to avoid confusion.

Reply:
or "random".